apps/codeforafrica/src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -0,0 +1,55 @@
+import { RichTypography } from "@commons-ui/core";
+import { Link } from "@commons-ui/next";
+import {
+  Card,
+  CardContent,
+  CardActions,
+  CardMedia,
+  Button,
+  Box,
+} from "@mui/material";
+import React from "react";
+
+const OpportunityCard = React.forwardRef(function OpportunityCard(props, ref) {
+  const { title, description, image, date, tags, comments, href, ...other } =
+    props;
+
+  if (!title && !description) {
```

Comment:
```suggestion
  if (!(title && description)) {
```

`!(A AND B) = !A OR !B` Boolean Algebra [anyone](https://en.wikipedia.org/wiki/De_Morgan%27s_laws)?